src/backends/custom/custom_backend_factory.cc

```diff
@@ -66,12 +66,20 @@ CustomBackendFactory::CreateBackend(
       GetDirectoryFiles(path, true /* skip_hidden_files */, &custom_files));
 
   std::unordered_map<std::string, std::string> custom_paths;
+  std::vector<std::shared_ptr<TemporaryDirectory>> local_custom(
+      custom_files.size());
+  for (size_t s = 0; s < custom_files.size(); s++) {
```

Comment:
You don't need a separate loop to initialize local_custom. Just above like 78 you can local_custom.push_back() and then pass &local_custom.back() into LocalIzeFileFolder() to initialize (see my comments below for LocalizeFileFolder.